guides/source/developers/events/overview.html.md

```diff
@@ -58,12 +59,12 @@ Spree::Event.subscribe /.*\.spree$/ do |event|
 end
 ```
 
-Please note that, unless you add explicitly the `.spree` suffix namespace,
-you will register to all ActiveSupportNotifications, including Rails internal
-ones.
+Please note that, when using a regexp, unless you add explicitly the `.spree`
+suffix namespace you will register to all ActiveSupportNotifications, including
+Rails internal ones.
 
-Another way to subscribe to events is creating a "subscriber" module that
-includes the `Spree::Event::Subscriber` module. For example:
+You can subscribe to events also by creating a "subscriber" module that includes
```

Comment:
This should be "You can _also_ subscribe to events _by_ creating a "subscriber" module".